Zend/zend_call_stack.c

```diff
@@ -119,11 +119,13 @@ static bool zend_call_stack_get_linux_pthread(zend_call_stack *stack)
 
 	error = pthread_getattr_np(pthread_self(), &attr);
 	if (error) {
+		pthread_attr_destroy(&attr);
```

Comment:
I think that we shouldn't call it in this case

Reply:
My first impression was that it does not hurt to call and that `pthread_getattr_np()` might have allocated already. Will double check and report back

Reply:
LOL, you are right. This call is not needed. In case the return value from `pthread_getattr_np()` would not be `0`, it does this call on it's own in the end.

Reply:
removed that call and pushed, thanks for the hint!